EasyPost/Models/API/CarrierAccountType.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections.Generic;
+using EasyPost.Utilities.Internal;
+
+namespace EasyPost.Models.API;
+
+public class CarrierAccountType : ValueEnum
+{
+    public static readonly CarrierAccountType AmazonMws = new CarrierAccountType(1, "AmazonMwsAccount");
```

Comment:
suggestion: I would not go down this path. Carriers on the EasyPost platform are constantly coming and going. This adds much more maintenance to the project than we probably want as we've previously discussed - keep the libs dumb.